test/util/IdTableHelpers.cpp

```diff
@@ -98,33 +104,19 @@ IdTable createRandomlyFilledIdTable(
   auto joinColumnGeneratorView = std::views::values(joinColumnWithGenerator);
 
   // Are all the join column numbers within the max column number?
-  if (auto column = std::ranges::find_if(
-          joinColumnNumberView,
-          [&numberColumns](const size_t& num) { return num >= numberColumns; });
-      column != joinColumnNumberView.end()) {
-    throw std::runtime_error(
-        absl::StrCat("The join columns", *column, " is out of bounds."));
-  }
+  AD_CONTRACT_CHECK(std::ranges::all_of(
+      joinColumnNumberView,
+      [&numberColumns](const size_t& num) { return num < numberColumns; }));
```

Comment:
Numbers can always be taken by value (and typically should) so just `size_t`

Reply:
Done, see [f3b6981](https://github.com/ad-freiburg/qlever/pull/1016/commits/f3b6981fb6712b4067519ee9078e10c0b64cb565).